bmcstored_dump_entry.cpp

```diff
@@ -86,7 +99,7 @@ void Entry::initiateOffload(std::string uri)
 
     if (pid > 0)
     {
-        Child::Callback callback = [this](Child&, const siginfo_t* si) {
+        Child::Callback callback = [this, uri](Child&, const siginfo_t* si) {
```

Comment:
what is this uri change for, if it is not for this commit, please make a different patch